gitpod-io/gitpod
# [ws-proxy] Reload to restart

This PR brings the "reload to restart" feature back to ws-proxy, cleans up the central `routes.go, makes unit tests for the routes easier and increases ws-proxy test coverage from 50% to > 60%.

### How to test
1. Start a workspace
2. Stop the workspace using the command palette
3. Once the workspace is stopped, reload the tab. The workspace should start again.
